packages/morph/components/editor.tsx

```diff
@@ -239,6 +240,30 @@ export default function Editor() {
     return extensions
   }, [settings.vimMode])
 
+   // Function to export Markdown file
+   const exportMarkdown = () => {
+    const blob = new Blob([markdownContent], { type: "text/markdown" });
+    const url = URL.createObjectURL(blob);
+    const a = document.createElement("a");
+    a.href = url;
+    // TODO: update the name based on users imported files.
+    a.download = "document.md";
+    document.body.appendChild(a);
+    a.click();
+    document.body.removeChild(a);
```

Comment:
we shouldn't do this, update to the DOM like this are very fragile. fwiw you should dump it to a new window as export.